CONTRIBUTING.md

```diff
@@ -1,12 +1,20 @@
+# How to contribute
+
+- Report bugs and suggestions in the [issues tab](https://github.com/Farseer-NMR/FarSeer-NMR/issues), use TAGS as appropriate.
+- **Always** submit a Pull Request from you Forked repository of Farseer-NMR.
```

Comment:
a fork isn't necessary. a clone would be better, since forks diverge, whereas clones stick together